src/main/java/org/jitsi/utils/collections/JMap.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.utils.collections;
+
+import java.util.*;
+
+/**
+ * Provides helpers for creating a Map easily in place.  Java 9 has
+ * these features under 'Map' but I scoped them under 'JMap' ('J'
+ * for 'Jitsi') to avoid conflicts with java.util.Map.
+ *
+ * Note that both styles--one taking a vararg of Entries and another
+ * taking individual arguments--are in Java 9 and are provided here
+ * for convenience.
+ */
+public class JMap
+{
+    /**
+     * Helper function to create a Map from a list of entries
+     *
+     * NOTE: without @SafeVarargs, we get a complaint about heap
```

Comment:
The Note about @SafeVarargs shouldn't be part of the Javadoc I don't think, unless it's relevant to the user rather than the implementor?

Reply:
i was worried about whether or not putting a comment below the javadoc comment would 'disassociate' the javadoc comment from the method itself for things trying to parse it (we don't actually do that, so i don't know for sure and couldn't test it).